Test/Test.fs

```diff
@@ -12,9 +12,11 @@ open FParsec
 open FParsec.Error
 open FParsec.Primitives
 
-exception TestFailed of string
+exception TestFailed of string * System.Diagnostics.StackTrace
 
-let Fail() = raise (TestFailed("Test failed."))
+let Fail() =
+    let st = System.Diagnostics.StackTrace()
+    raise (TestFailed("Test failed.", st))
```

Comment:
Exceptions contain a stack trace on their own, why add it again?

Reply:
🤦